src/v/model/transform.h

```diff
@@ -70,6 +70,12 @@ using transform_name = named_type<ss::sstring, struct transform_name_tag>;
 using transform_name_view
   = named_type<std::string_view, struct transform_name_view_tag>;
 
+/**
+ * Whether a transform is or should be paused (i.e. stopped but not removed from
+ * the system).
+ */
+using is_paused = ss::bool_class<struct is_paused_tag>;
```

Comment:
Should we `is_transform_paused`? Honestly I regret not adding a nested `transform` namespace here, everything is prefixed with `transform` so it's become a "namespace" in the C sense :sigh:

Reply:
done